src/MxcImageProvider.cpp

```diff
@@ -120,7 +125,13 @@ MxcImageProvider::download(const QString &id,
         if (fileInfo.exists()) {
             QImage image = utils::readImageFromFile(fileInfo.absoluteFilePath());
             if (!image.isNull()) {
-                image = image.scaled(requestedSize, Qt::KeepAspectRatio, Qt::SmoothTransformation);
+                auto size = requestedSize;
+                if (size.width() == 0) {
```

Comment:
I think you should just use scaledToHeight in this case.

Reply:
Makes sense, didn't know that existed. 😊

Reply:
We have nice things here